test/PasskeyAccount.t.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: UNLICENSED
+// Force a specific Solidity version for reproducibility.
+pragma solidity ^0.8.13;
+
+import { Test } from "forge-std/Test.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { console2 } from "forge-std/console2.sol";
+
+import { EntryPoint } from "@aa/core/EntryPoint.sol";
+import { IEntryPoint } from "@aa/interfaces/IEntryPoint.sol";
+import {
+    UserOperation, UserOperationLib
+} from "@aa/interfaces/UserOperation.sol";
+
+import { Create2 } from "@openzeppelin/contracts/utils/Create2.sol";
+
+import { PasskeyAccount, Base64Url } from "src/account/PasskeyAccount.sol";
+import { PasskeyAccountFactory } from "src/account/PasskeyAccountFactory.sol";
+
+contract PasskeyAccountTest is Test {
+    using UserOperationLib for UserOperation;
+
+    address constant entryPointMainnetAddr =
+        0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789;
+    address constant entryPointGethAddr =
+        0x5FbDB2315678afecb367f032d93F642f64180aa3;
+    string constant credId = "9h5F3DgLSjSMdnVOadmhCw";
+    uint256 constant pubKeyX =
+        67299174900712686363169673082376821529726602378544032702281553676098545184711;
+    uint256 constant pubKeyY =
+        104273800132786176334597151467609377740095818152192999025225464410568038480397;
+    uint256 constant salt = 0;
+
+    IEntryPoint public entryPoint;
+    PasskeyAccountFactory public passkeyAccountFactory;
+    PasskeyAccount public passkeyAccount;
+
+    function setUp() public {
+        // Align the entryPoint address with the one on the Gethnode and Mainnet
+        bytes memory entryPointBytecode =
+            vm.getCode("EntryPoint.sol:EntryPoint");
+        address entryPointAnotherAddr =
+            Create2.deploy(0, bytes32(uint256(2)), entryPointBytecode);
+        vm.etch(entryPointGethAddr, entryPointAnotherAddr.code);
+        vm.etch(entryPointMainnetAddr, entryPointAnotherAddr.code);
```

Comment:
Maybe we don't need `entryPointAnotherAddr`? The second argument of `vm.etch` is bytecode, you may try to use `entryPointBytecode` directly

Reply:
And just choose the geth address (and rename it to `entryPointAddr`), we don't need to be aware chain information in test.